.github/workflows/testing.yml

```diff
@@ -0,0 +1,52 @@
+name: Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  behat:
+    strategy:
+      matrix:
+        php-version: ['8.2', '8.0', '7.4']
```

Comment:
Love to see the addition of these behat tests! 

Any insight into the matrix here? (why not bothering to test php 8.1/8.3)



Version | Usage Percentage (Jan 2025) | Active Support End | Security Support End
-- | -- | -- | --
7.4 | 7.6% | Nov 28, 2021 | Nov 28, 2022
8.0 | 3.4% | Nov 26, 2022 | Nov 26, 2023
8.1 | 18.1% | Nov 25, 2023 | Dec 31, 2025
8.2 | 28.6% | Dec 31, 2024 | Dec 31, 2026
8.3 | 32.7% | Dec 31, 2025 | Dec 31, 2027

Reply:
I don't have _the_ answer for you; I copied what core WP-CLI does for the sake of expedience. I agree, though, that we can mix this up a bit more. I think I'll do 8.3, 8.2, 7.4.